internal/pkg/service/stream/keboolasink/bridge/table.go

```diff
@@ -102,3 +107,25 @@ func (b *Bridge) createTable(ctx context.Context, tableKey keboola.TableKey, col
 	b.logger.Info(ctx, "created table")
 	return tab, nil
 }
+
+func (b *Bridge) addTableMetadata(ctx context.Context, table *keboola.Table, sink definition.Sink) error {
+	api, err := b.apiProvider.APIFromContext(ctx)
+	if err != nil {
+		return err
+	}
+
+	for i := range table.Metadata {
+		if table.Metadata[i].Key == exportMetaKey {
+			return nil
+		}
+	}
```

Comment:
Just to be sure, did you create task for potentional optimization when all existing tables will be filled with metadata and this code will be executed only after `CreateTable` call? :slightly_smiling_face: